src/App.css

```diff
@@ -0,0 +1,95 @@
+html,
+body {
+  height: 100%;
+}
+body {
+  background-color: var(--background-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  align-items: center;
+  gap: 20px;
+  max-width: 500px;
+  background-color: var(--card-bg-color);
+  border-radius: 10px;
+  padding: 20px;
+}
+
+.title {
+  color: var(--neon-green);
+  font-size: 16px;
+  letter-spacing: 3px;
+}
+
+.advice {
+  color: var(--text-color);
+  text-align: center;
+  font-size: 25px;
+}
+
+.divider {
+  width: 80%;
+}
+
+.dice-icon {
+  width: 60px;
+  height: 60px;
+  background-color: var(--neon-green);
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: -50px;
+}
+
+.dice-icon:hover {
+  box-shadow: 0 0 10px 2px var(--neon-green);
+  cursor: pointer;
+}
+
+.rotate {
+  animation: rotateAnimation 1s infinite linear;
+}
+
+@keyframes rotateAnimation {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(180deg);
+  }
+}
+
+@media screen and (max-width: 900px) {
+  .title {
+    font-size: 25px;
+  }
+
+  .advice {
+    font-size: 35px;
+  }
+
+  .card {
+    min-height: 300px;
+  }
+}
+
+@media screen and (max-width: 576px) {
+  .title {
+    font-size: 18px;
+  }
+
+  .advice {
+    font-size: 30px;
+  }
+  .card {
+    min-height: 300px;
+    max-width: 300px;
+  }
+}
```

Comment:
This works, but your approach in `rating-system` is better, 

Two separate `@media` with a lot of changes are not really optimal in this case, as you can do it with 1 or 0 `@media` queries,

Consider using responsive approaches like `flex`, `%`, `max-`, `min-`, `clamp`, etc.